validator/client/validator.go

```diff
@@ -128,7 +134,105 @@ type attSelectionKey struct {
 
 // Done cleans up the validator.
 func (v *validator) Done() {
-	v.ticker.Done()
+	if v.accountChangedSub != nil {
```

Comment:
not sure if we need to unsubscribe here, there are other channels we don't close such as eventsChannel/accountsChangedChannel ( this is because of how our health check design is done, we need to improve this in a future PR)